torchvision/prototype/transforms/functional/__init__.py

```diff
@@ -82,3 +82,25 @@
     to_image_tensor,
     to_image_pil,
 )
+
+
```

Comment:
We put the class to bundle the functions together on `functional/__init__.py` for convenience and minimum changes for now. Later to make less writing, I think the import on __init__.py need to be changes into something like: `import ._geometry as geometry`